omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -271,6 +234,10 @@ def __init__(self, task, scene, robot, teleport=False):
         self.robot_model = self.robot.model_name
         self.robot_base_mass = self.robot._links["base_link"].mass
         self.teleport = teleport
+
+        self.robot_copy = RobotCopy()
+        self._load_robot_copy()
```

Comment:
`self.robot_copy = load_robot_copy(robot)`

i.e. let's make load_robot_copy into a static function and make it return a robot copy.